scripts/openshift-tools-scripts.spec

```diff
@@ -190,6 +191,7 @@ OpenShift Tools Docker Monitoring Scripts
 /usr/bin/cron-send-docker-containers-usage
 /usr/bin/cron-send-docker-dns-resolution
 /usr/bin/cron-send-docker-existing-dns-resolution
+/usr/bin/cron-send-docker-oc-versions
 
```

Comment:
@twiest Currently added to docker rpm, and tested locally with tito. If you want it move to openshift I'll move it.